src/js/pages/system/UnitsHealthDetail.js

```diff
@@ -47,6 +55,24 @@ class UnitsHealthDetail extends mixin(StoreMixin) {
     UnitHealthStore.fetchUnitNodes(this.props.params.unitID);
   }
 
+  onUnitHealthStoreUnitSuccess() {
+    let isLoading = this.state.isLoading & ~FLAGS.UNIT_LOADED;
```

Comment:
So a Util that you would just pass a variable `isLoading` and the bits that you want to turn off?